store/tikv/unionstore/memdb_norace_test.go

```diff
@@ -50,11 +51,13 @@ func (s testMemDBSuite) TestRandom(c *C) {
 		op := rand.Float64()
 		if op < 0.35 {
 			p1.DeleteKey(k)
-			p2.Delete(k)
+			// to fix: delete key not found
+			_ = p2.Delete(k)
```

Comment:
@tiancaiamao 
I think we need to check this error.

Reply:
I think it does not need to be so strict, there is already `_ = p2.Put(k, k)` in the previous code. And it is a test case.